etc/rules/0015-ossec_rules.xml

```diff
@@ -388,8 +388,8 @@
   </rule>
 
   <rule id="700" level="0">
-    <category>ossec</category>
-    <decoded_as>ossec-logcollector</decoded_as>
+    <category>wazuh</category>
+    <decoded_as>wazuh-logcollector</decoded_as>
```

Comment:
Same question as in the ruleset repository. This already changed?

Reply:
The _wazuh-logcollector_ decoder was already added, replacing _ossec-logcollector_.